source/about/licensing-and-subscription.rst

```diff
@@ -1,125 +1,31 @@
-Licensing and Subscription
```

Comment:
Given that the title of this existing docs page has changed, let's also change the filename of the file itself to match, and add a redirect from the existing page to the new page.

Reply:
Updated to just 'subscription.rst'

Reply:
Thanks! Please also update the LHS link to the updated file via ``overview/index.html`` like this:

```
.. toctree::
   :maxdepth: 2
   :glob:

   /about/product
   ...
   /about/subscription
```

Then, add the following redirect link in ``conf.py``, under the ``# Redirects section``

```
"about/licensing-and-subscription": "https://docs/mattermost.com/about/subscription.html",
```

Feel free to make this the very first redirect if you like since it starts with "a".

Reply:
Added and updated